copter/source/docs/traditional-helicopter-tips.rst

```diff
@@ -0,0 +1,22 @@
+.. _traditional-helicopter-tips:
+
+===========================
+Traditional Helicopter Tips
+===========================
+
+- Be sure to follow the wiki instructions, step by step!
+- Be sure the CG is in line with the main rotor shaft
+- Blade tracking must be within a quarter blade width or better to avoid excessive vibration
+- Vibration is the arch enemy of Helicopter autopilots!
+   - Internal Combustion Engines and Turbines can created high frequency vibrations that do not appear in the VIBE logs and can be aliased to low frequencies which can impact EKF and Attitude control (see below for ground tests to check for this). This also can occur in electric helicopters.
+   - The autopilot should be well isolated for vibration. Using autopilots with internal IMU vibration damping and vibration reducing mounts is recommended.
+- Traditional Helicopter *is* capable of inverted flight and using full negative collective ranges.
+
+Checking for High Frequency Vibration
+=====================================
+
+Symptoms of high frequency vibration can be that the attitude estimate becomes incorrect causing it to lean significantly even though the pilot is commanding level flight in stabilized modes. In the extreme, it can result in an :ref:`ekf-inav-failsafe` or even a crash.
+
+One method of testing this, is to secure the helicopter to the ground, and run up the main rotor, watching the artificial horizon in the ground control station. If it moves away from level, you probably have a vibration issue which needs to be addressed. Testing should be done at various collective levels, and at whatever rotor speeds will be used. Be careful not to over-stress the helicopter with collective extremes.
```

Comment:
yep, I only hold them down to check initial tracking, but Tridge uses this to test for high freq aliasing and suggested it, with the understanding that it could have ground resonance....I have never done this myself as a test for it, only had current build go wonky in the air due to bad tracking before I tracked them....in my filter testing, I have found that the high frequency aliasing does appear in the batch sampler plots....we probably should discuss with @tridge

Reply:
he said on the ground since it gives one "true" reference,ie you want it level earth frame...if the horizon rolls you have high freq vibs...which I am not sure I agree with...low freq vibs or aliased high freq vibs screw up attiutde, while only high freq vibs screw up the EKF Tridge said....
as to aliasing, a picture is easiest to show aliasing:
![prefilter](https://user-images.githubusercontent.com/6076534/120017872-01c68680-bfac-11eb-8e8b-aca72e1329a8.png)

Reply:
updated based on our chat with Tridge last night...